library/core/src/num/dec2flt/common.rs

```diff
@@ -0,0 +1,198 @@
+//! Common utilities, for internal use only.
```

Comment:
Nit (maybe for a follow-up?): `pub (crate)` is a perfect way to document and enforce internal use.

Reply:
Ah great idea, thank you. Resolved as of the latest commit.